yt/yt/server/controller_agent/controllers/competitive_job_manager.h

```diff
@@ -30,7 +30,37 @@ struct ICompetitiveJobManagerHost
 
 ////////////////////////////////////////////////////////////////////////////////
 
+class IExtraJobManager
+{
+public:
+    IExtraJobManager() = default;
+
+    virtual i64 GetPendingCandidatesDataWeight() const = 0;
+
+    virtual int GetPendingJobCount() const = 0;
+
+    virtual int GetTotalJobCount() const = 0;
+
+    //! returns if the cookie should be returned back to the pool
```

Comment:
Returns 'true' ?

And please write the commends from the capital letter and do not forget to add dot at the end of the sentence.